modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -4,13 +4,17 @@
 // * installing/installing_vmc/installing-vmc-network-customizations.adoc
 // * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
 // * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
 
 ifeval::["{context}" == "installing-vsphere-installer-provisioned-network-customizations"]
 :network:
 endif::[]
 ifeval::["{context}" == "installing-vmc-network-customizations"]
 :network:
 endif::[]
+ifeval::["{context}" == "installing-restricted-networks-installer-provisioned-vsphere"]
+:restricted:
+endif::[]
 
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
 = Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
```

Comment:
I see that this is an existing module, but per the OpenShift guidelines: Do not use backticks or other markup in headings.